### PR TITLE
Astro 10/11 release notes

### PR DESCRIPTION
**Pull request by jwitz**
Based on https://app.zenhub.com/workspaces/developer-productivity-and-readiness-6242176c89ed9000197ec734/reports/release?release=Z2lkOi8vcmFwdG9yL1JlbGVhc2UvODMyMjY

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -15,6 +15,27 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.6.0 ([Release notes](cli/release-notes.md))
 
+## October 11, 2022 
+
+### Improved lineage tab in the Cloud UI
+
+The **Lineage** tab has been updated with both a more intuitive user experience and a new look and feel to match the Astronomer Cloud UI. Specifically:
```
Comment (kwcanuck) [inline astro/release-notes.md:22]:
```suggestion
The following changes have been made to the **Lineage** tab in the Cloud UI to provide a more intuitive user experience: 
```

File: astro/release-notes.md [hunk 2]
```diff
@@ -15,6 +15,27 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.6.0 ([Release notes](cli/release-notes.md))
 
+## October 11, 2022 
+
+### Improved lineage tab in the Cloud UI
+
+The **Lineage** tab has been updated with both a more intuitive user experience and a new look and feel to match the Astronomer Cloud UI. Specifically: 
+
+- Opening the **Lineage** tab no longer refreshes your web browser, nor does it change the design of surrounding tabs.
```
Comment (kwcanuck) [inline astro/release-notes.md:24]:
```suggestion
- Opening the **Lineage** tab no longer refreshes your browser or changes the appearance of the surrounding tabs.
```

File: astro/release-notes.md [hunk 3]
```diff
@@ -15,6 +15,27 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+- Then you click a run in the lineage graph, the lineage path through the run is now animated to help illustrate the flow of data through your pipeline.
```
Comment (kwcanuck) [inline astro/release-notes.md:25]:
```suggestion
- When you click a run in the lineage graph, the lineage path through the run is now animated to help illustrate the flow of data through your pipeline.
```

File: astro/release-notes.md [hunk 4]
```diff
@@ -15,6 +15,27 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+- The **Compare** feature is now easier to use, with new visual icons and color changes to help show which two runs are being compared.
```
Comment (kwcanuck) [inline astro/release-notes.md:26]:
```suggestion
- Icons and color have been added to the **Compare** feature to make it easier to identify the individual runs being compared.
```

File: astro/release-notes.md [hunk 5]
```diff
@@ -15,6 +15,27 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+- You can now search for runs and datasets directly from the **Runs** and **Datasets** page respectively..
```
Comment (kwcanuck) [inline astro/release-notes.md:27]:
```suggestion
- You can now search for runs and datasets directly in the **Runs** and **Datasets** pages.
```

File: astro/release-notes.md [hunk 6]
```diff
@@ -15,6 +15,27 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+![New lineage tab](/img/release-notes/updated-lineage.png)
+
+### Additional improvements 
+
+- New worker node pools on Azure and Google Cloud Platform (GCP) clusters can now scale to zero. This means that enabling a new worker type for your cluster does not cost you until it's used in a Deployment.
```
Comment (kwcanuck) [inline astro/release-notes.md:33]:
```suggestion
- New worker node pools on Azure and Google Cloud Platform (GCP) clusters can now scale to zero.  You don't incur costs for enabling a new worker type for your cluster until it's used in a Deployment.
```

File: astro/release-notes.md [hunk 7]
```diff
@@ -15,6 +15,27 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+
+### Bug fixes 
+
+- Fixed an issue where worker queues with a minimum worker count of 0 would appear with a minimum worker count of 1 in the Cloud UI.
```
Comment (kwcanuck) [inline astro/release-notes.md:37]:
```suggestion
- Fixed an issue where worker queues with a minimum worker count of zero would appear with a minimum worker count of one in the Cloud UI.
```

**Review by kwcanuck — APPROVED** [review]

**Review by amccabe-astronomer — APPROVED** [review]
No issues from my perspective

File: astro/release-notes.md [hunk 8]
```diff
@@ -15,6 +15,27 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+### Improved lineage tab in the Cloud UI
+
+The following changes have been made to the **Lineage** tab in the Cloud UI to provide a more intuitive user experience: 
+
+- Opening the **Lineage** tab no longer refreshes your browser or changes the appearance of the surrounding tabs.
+- When you click a run in the lineage graph, the lineage path through the run is now animated to illustrate the flow of data through your pipeline.
+- Icons and color have been added to the **Compare** feature to make it easier to identify the individual runs being compared.
+- You can now search for runs and datasets directly in the **Runs** and **Datasets** pages.
+
+![New lineage tab](/img/release-notes/updated-lineage.png)
```
Comment (jwitz) [inline astro/release-notes.md:29]:
```suggestion
```

**Pull request merged**